### PR TITLE
Fix dependencies of openQA-client package

### PR DESCRIPTION
**Pull request by Martchus**
* Move all client specific files into openQA-client itself
* Make it depend on openQA-common because is requires
  OpenQA::File

---

<s>Not tested yet</s>

### CODE REVIEW

File: openQA.spec [hunk 1]
```diff
@@ -451,6 +453,7 @@ fi
 %{_datadir}/openqa/script/load_templates
 %dir %{_datadir}/openqa/lib
 %{_datadir}/openqa/lib/OpenQA/Client.pm
+%{_datadir}/openqa/lib/OpenQA/Client/*
```
Comment (coolo) [inline openQA.spec]: Did you test if this builds? I *think* you need to remove the /* at the end - but please double check

Comment (Martchus) [reply openQA.spec]: > Not tested yet

**Comment by Martchus** [comment]
I tested it by running `osc build` locally. The `/*` at the end needed to be removed indeed. Otherwise the directory itself isn't excluded from openQA-common.

**Comment by okurz** [comment]
@Martchus `osc build` will only care about the "BuildRequires" which are buildtime dependencies. The runtime dependencies are specified by "Requires" and this is what you changed. So you would need to follow with what openQA-test.spec is doing but for the client package.

**Comment by coolo** [comment]
The goal was to check if it builds - and it didn't in his first submission

**Pull request merged**